libconcurrent/includes/primitives.h

```diff
@@ -77,6 +79,8 @@ inline uint64_t synchNonZeroBits(uint64_t v);
 #    define synchPause()                                                                                                                                                                               \
         {                                                                                                                                                                                              \
             int __i;                                                                                                                                                                                   \
+            if (synchGetMachineModel() != INTEL_X86_MACHINE)                                                                                                                                           \
+                return;                                                                                                                                                                                \
```

Comment:
Why that?

Reply:
It seems that newest Intel X86 machines behave much better on spinning loops without executing any `pause` operation.

Reply:
then a comment will be useful in the code here :-)

Reply:
Check the e7df25c commit.